nats/src/jetstream/types.rs

```diff
@@ -708,11 +710,30 @@ pub struct ConsumerInfo {
     pub push_bound: bool,
 }
 
-/// Information about the consumer's associated `JetStream` cluster
+/// Information about the stream's, consumer's associated `JetStream` cluster
 #[derive(Debug, Default, Serialize, Deserialize, Clone, PartialEq, Eq)]
 pub struct ClusterInfo {
-    /// The leader of the cluster
+    /// The cluster name.
+    pub name: String,
+    /// The server name of the RAFT leader.
     pub leader: String,
+    /// The members of the RAFT cluster.
+    pub replicas: Vec<PeerInfo>,
+}
+
+/// The members of the RAFT cluster
+#[derive(Debug, Default, Serialize, Deserialize, Clone, PartialEq, Eq)]
+pub struct PeerInfo {
+    /// The server name of the peer.
+    pub name: String,
+    /// Indicates if the server is up to date and synchronised.
+    pub current: bool,
+    /// Nanoseconds since this peer was last seen.
+    pub active: usize,
+    /// Indicates the node is considered offline by the group.
+    pub offline: Option<bool>,
```

Comment:
Optional bool, so could probably be collapsed to false by default.

```suggestion
    #[serde(default)]
    pub offline: bool,
```

Reply:
Fix it !